.github/workflows/ci.yml

```
@@ -17,15 +17,10 @@ jobs:
   test:
       strategy:
         fail-fast: false
-        matrix:
-          os: [ubuntu-latest]
-          crystal: [latest, nightly]
-      runs-on: ${{ matrix.os }}
+      runs-on: ubuntu-latest
+      container:
+        image: crystallang/crystal
```

This should allow more coverage to see if we have upcoming breaking changes

```suggestion
        matrix:
          crystal:
          - latest
          - nightly
          - 1.0.0
          - 0.36.1
      runs-on: ubuntu-latest
      container:
        image: crystallang/crystal:${{ matrix.crystal }}
```